lib/pure/random.nim

```diff
@@ -7,9 +7,9 @@
 #    distribution, for details about the copyright.
 #
 
-## Nim's standard random number generator.
+## Nim's standard random number generator (RNG).
```

Comment:
now that you've defined RNG in a very visible place on top, how about using RNG everywhere in place of `random number generator` ?

Reply:
I didn't want to exaggerate using it (in fact I only used it in one play), but sure, I can do that.